core/src/klio_core/options.py

```diff
@@ -67,19 +67,30 @@ def handle_parse_result(self, ctx, opts, args):
         )
 
 
-def _verify_gcs_uri(ctx, param, value):
-    if value and not value.startswith("gs://"):
-        raise click.BadParameter(
-            "Unsupported location type. Please provide a GCS location with "
-            "the `gs://` prefix."
-        )
+def override(func):
```

Comment:
In the future, please maintain [atomic commits](https://www.freshconsulting.com/atomic-commits/) - this function and the `template` function are not relevant to the purpose of this particular commit (c8248988).